.releaserc

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
it's not enough to just add this config. The CI must be updated as well as repository secrets. It would be a better idea to do it in a separate PR